srcpkgs/udis86/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'udis86'
 pkgname=udis86
 version=1.7.2
-revision=3
+revision=4
 build_style=gnu-configure
-hostmakedepends="python"
 configure_args="--enable-shared"
+hostmakedepends="python"
 short_desc="Disassembler Library for x86 and x86-64"
 maintainer="Juan RP <xtraeme@voidlinux.org>"
 license="BSD"
```

Comment:
spdx short-identifier

Reply:
We should at a SPDX License list to xlint to catch these. I rarely check the License of old packages.

Reply:
The SPDX license list is massive

Reply:
Yep, I'm aware of that. Alpine is afaik linting agianst the full list.